src/components/Footer.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+const Footer = props => (
+  <footer className="footer">
+    <span className="todo-count">
+      <strong>{props.countActiveItems.length}</strong>
+      {' '}
+        item left
+    </span>
+    <ul className="filters">
+      <li>
+        <a
+          href="#/"
+          className={props.showSelectedList === 0 ? 'selected' : ''}
```

Comment:
Suggestion: pass constant `FILTER` as props of this component and use its values here instead of numbers 0, 1, 2